Neos.Flow/Configuration/Settings.Persistence.yaml

```diff
@@ -30,7 +30,7 @@ Neos:
         dbal:
           mappingTypes:
             'flow_json_array':
-              dbType: 'json_array'
+              dbType: 'json'
```

Comment:
i guess `json_array` was removed as type?

Reply:
Yep correct!

Reply:
interestingly i cant find a mention in the upgrade guide?  https://www.doctrine-project.org/2020/11/17/dbal-3.0.0.html

Reply:
Sadly not, no, it was juist deprecated and is now gone

Reply:
oke do you suspect this is breaking for our users or similar sneakily gone things that we stumbled upon but which are not documented which we should document then?

Reply:
That was the only really nasty one I have encountered, but I guess everyone using a lot of DBAL wil have to check their application